torch/_torch_docs.py

```diff
@@ -3612,6 +3612,34 @@ def merge_dicts(*dicts):
              [  2.0774,  -0.8187]]])
 """.format(**common_args))
 
+add_docstr(torch.matrix_exp,
+           r"""
+matrix_power(input) -> Tensor
+
+Returns the matrix exponential. Supports batched input.
```

Comment:
Comparing these docs with https://docs.scipy.org/doc/scipy/reference/generated/scipy.linalg.expm.html I would suggest citing the algorithm we're using. Also, it would also probably be a good idea to (1) provide a LaTeX formula for the mathematical meaning of matrix exponential and (2) remind the user that matrix exponential is not the same thing as matrix power.